test/trn_func_tests/test_delete_one_at_a_time.py

```diff
@@ -94,8 +94,8 @@ def setUp(self):
 
         # Delete an endpoint in subnet, delete the parent subnet, then parent VPC.
         self.ep3 = c.delete_simple_endpoint(3, 20, "10.20.0.4", "d3")
-        c.delete_network(3, 20, cidr("24", "10.20.0.0"), ["switch-2"])
-        c.delete_vpc(3, cidr("16", "10.0.0.0"), ["router-1"])
+        c.delete_network(3, 20, cidr("24", "10.20.0.0"))
```

Comment:
Why do you need the cidr to call delete_network?

Reply:
I don't. Fixing now.